e2e/tests/nightly/runtime.spec.ts

```diff
@@ -27,4 +43,46 @@ describe('Updating runtime status', () => {
     // Delete runtime
     await runtimePanel.deleteRuntime();
   });
+
+  test('Create with detachable disk, re-attach, delete', async () => {
```

Comment:
nit: If we make this new test in its own test file, it should helps CI nightly job finish sooner. Because test creates a new workspace, not reusing one created already in previous test.

Reply:
Done